docs/tutorials/configure/build-a-mock-robot.md

```diff
@@ -134,20 +149,20 @@ go run index.go
 If you successfully configured your robot and it is able to connect to the Viam app you should see something like this printed to the terminal after running your program.
 What you see here is a list of the various resources (Like components, and services) that have been configured to your robot in the Viam app.
 
-<img src="../../img/build-a-mock-robot/image1.png" alt="A screenshot from the Visual Studio Code command line that prints the output of print(robot.resource_names) when your Raspberry Pi has correctly connected and initialized with the Viam app. The output is an array of resources that have been pulled from the Viam app. Some of these are the Vision Service, Data Manager, and Board." width="100%">
+![Command line output from running python3 index.py when your Raspberry Pi has correctly connected and initialized with the Viam app. The output is an array of resources that have been pulled from the Viam app. The list includes the Motion Service, arm component, data manager, board component and motor component. There is also a list of arm position and orientation values.](../../img/build-a-mock-robot/resource-output.png)
 
-### How to control your mock robot
+### Control your mock robot
 
-Next, you will be writing some code to control and move your mock robotic arm.
-We are going to write a program that will move the mock robotic arm into a new random position every second.
-You will be able to verify that your mock robotic arm is working by checking that the joint positions of the fake arm in the **control** tab of the Viam app are changing.
+Next, you will write some code to control and move your mock robotic arm.
```

Comment:
I'm wondering whether it would be better to either change these statements to imperative (`Next, write...`) or use `can` instead (`Next, you can`). I feel like that leaves more agency with the reader?

Reply:
That's fine I think